src/Sql/Sql/ChangeLog.md

```diff
@@ -27,6 +27,7 @@
 ## Version 4.5.0
 * Added a new cmdlet to refresh external governance status
     - 'Invoke-AzSqlServerExternalGovernanceStatusRefresh'
+* Added `PreferredEnclaveType` parameter to `NewAzureSqlElasticPool`, `Get-AzSqlElasticPool` and `SetAzureSqlElasticPool` cmdlet
```

Comment:
Is `PreferredEnclaveType added to `Get-AzSqlElasticPool`? Not find in codes

Reply:
> oh, so doing a get call on elastic pool with `Get-AzSqlElasticPool` now gives out the new parameter `PreferredEnclaveType` as well, but there are no changes made to the get cmdlet as such to support this change.

What do you mean by “Added `PreferredEnclaveType` parameter `Get-AzSqlElasticPool` cmdlet”?

Reply:
updated the ChangeLog.md